control-plane-operator/controllers/hostedcontrolplane/oapi/config.go

```diff
@@ -89,3 +91,34 @@ func reconcileConfigObject(cfg *openshiftcpv1.OpenShiftAPIServerConfig, etcdURL,
 		},
 	}
 }
+
+const haproxyTemplate = `
+global
+  maxconn 7000
+  log stdout local0
+  log stdout local1 notice
+
+defaults
+  mode http
+  log global
+  option log-health-checks
+  balance roundrobin
+  timeout client 10m
+  timeout server 10m
+  timeout connect 10s
+  timeout client-fin 5s
+  timeout server-fin 5s
+  timeout queue 5s
+  retries 3
+
+frontend http-in
+  bind *:10080
+  default_backend be
+backend be
+  mode http
+  server ks konnectivity-server-local:8090 ssl verify required ca-file /etc/konnectivity-proxy-tls/ca.crt crt /tmp/tls.pem
```

Comment:
get value for the local konnectivity service name from the manifests package,
use `volumeMounts` for path to ca.crt
use a constant for the cert path here and in the script that creates it.

Reply:
done